### PR TITLE
Add `advertise-address` to config

### PR DESCRIPTION
**Pull request by crazycs520**
## What have you changed? (mandatory)

* This PR add `advertise-address` configuration to config.
* `advertise-address` is the current TiDB server IP. We need put some information such as IP to PD.This is for issue[#6961](https://github.com/pingcap/tidb/issues/6961)
* Why do not use `host` of the config directly? Because the `host` Maybe "0.0.0.0"

## Does this PR affect documentation (docs/docs-cn) update? (mandatory)
Yes.

## Does this PR affect tidb-ansible update? (mandatory)
Yes.

### CODE REVIEW

**Comment by crazycs520** [comment]
@winkyao @zimulala @coocood PTAL

**Comment by shenli** [comment]
Please add proper labels.

**Comment by shenli** [comment]
This PR definitely affect documentation.

**Comment by shenli** [comment]
This is still "No":
> Does this PR affect documentation (docs/docs-cn) update? (mandatory)
No

File: config/config.toml.example [hunk 1]
```diff
@@ -3,6 +3,9 @@
 # TiDB server host.
 host = "0.0.0.0"
 
+# tidb server advertise IP.
+advertise-ip = ""
```
Comment (coocood) [inline config/config.toml.example]: IP or address?

**Comment by winkyao** [comment]
@zz-jason This PR is prework of #6961, we can add the release-note label to next PR.

File: config/config.toml.example [hunk 2]
```diff
@@ -3,6 +3,9 @@
 # TiDB server host.
 host = "0.0.0.0"
 
+# tidb server advertise IP.
```
Comment (shenli) [inline config/config.toml.example]: Will the address contain port? Or only IP?

Comment (crazycs520) [reply config/config.toml.example]: Only IP.

**Comment by shenli** [comment]
LGTM
Please send PRs to tidb-ansible, docs-cn and docs.

File: config/config.toml.example (hunk 2 above)
Comment (coocood) [reply config/config.toml.example]: Can we obtain the IP from system information?

**Pull request merged**